endoreg_db/serializers/label_video_segment/label_video_segment_update.py

```
@@ -0,0 +1,182 @@
+from endoreg_db.models import LabelVideoSegment, VideoPredictionMeta
+from endoreg_db.serializers.video.segmentation import LabelSegmentSerializer
```

Pylint (3.3.7)

[error] No name 'LabelSegmentSerializer' in module 'endoreg_db.serializers.video.segmentation'

(E0611)